src/scripts/main.js

```diff
@@ -354,7 +354,19 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dashboard = document.querySelector('.dashboard');
 
-// write your code here
+const markup = people
+  .map((person) => {
+    return `<tr>
+      <td>${person.name}</td>
+      <td>${person.sex}</td>
+      <td>${person.born}</td>
+      <td>${person.died}</td>
+      <td>${person.died - person.born}</td>
+      <td>${Math.ceil(person.died / 100)}</td>
```

Comment:
The calculation for the century a person died in might not be accurate. The current formula `Math.ceil(person.died / 100)` will incorrectly assign the year 2000 to the 21st century instead of the 20th. Consider adjusting the formula to correctly handle edge cases, such as using `Math.ceil((person.died - 1) / 100)`.